kernel/packages/unity-interface/UnityParcelScene.ts

```diff
@@ -59,7 +60,15 @@ export class UnityPortableExperienceScene extends UnityScene<LoadablePortableExp
     this.worker
       .getAPIInstance(ParcelIdentity)
       .then((parcelIdentity) => {
+        parcelIdentity.land = {
+          baseUrl: this.data.data.land!.baseUrl,
+          mappingsResponse: this.data.data.land!.mappingsResponse,
+          baseUrlBundles: this.data.data.land!.baseUrlBundles,
+          sceneId: this.data.data.land!.cid,
+          sceneJsonData: this.data.data.land!.sceneJsonData
+        }
```

Comment:
We should find another mechanism to do this, since PX are not tied to any LAND

Reply:
Yeah, take a look to my last commited changes. With the approach of creating a `isPortableExperience` flag that you suggested me, it is not already necessary to set a land for a PE parcel identity.